### ISSUE TITLE
loss problem.

### ISSUE THREAD

**Issue by chuanruihu** [timctho/unet-pytorch#1]
hi, I want to know what is the function of the dice_loss ? The code was used in run_train.py line11. The U-Net paper equation (2) is how to  exhibite in your code? thank you.

**Comment by timctho** [comment]
Dice loss is to minimize f1 score. Some introduction here.
[https://dev.to/andys0975/what-is-dice-loss-for-image-segmentation-3p85](url)

**Comment by chuanruihu** [comment]
Thank you for your reply. These days I am learning U_Net to segmentate cell. When I use your code to carry out it. I find your code don't have equation (2) in U-net peper. Can you tell me why?

**Comment by timctho** [comment]
It is a pre-computed weight map for each training image, to enforce the loss on edge.
In my unet-tensorflow repo I add the weighting, but no big difference on my task.

**Comment by chuanruihu** [comment]
Thank you. I just find the code about loss on the edge  in your u-net-tf. I don't find your code . Can you give me a direct  position in your code??

**Comment by timctho** [comment]
You can see [https://github.com/timctho/unet-tensorflow/blob/master/model/u_net_tf_v2.py](url)
From line 157, provide a pre-computed weight map, and multiply loss on it.
